motr/m0crate/crate.c

```diff
@@ -1254,6 +1255,28 @@ static void cr_btree_delete(struct m0_key_val *kv)
 			  bo_u.u_btree.t_rc);
 }
 
+static void cr_btree_warmup(int count, int max_ksize, int max_vsize)
+{
+	int i = 0;
+
+	for (i = 0; i < count ; i++)
+	{
+		struct m0_key_val	dummy_kv;
+		int			ksize = getrnd(1, max_ksize);
+		int			vsize = getrnd(1, max_vsize);
+		char			k[max_ksize];
+		char			v[max_vsize];
+
+		m0_buf_init(&dummy_kv.kv_key, k, ksize);
+		m0_buf_init(&dummy_kv.kv_val, v, vsize);
+
+		sprintf(k, "%0*d", ksize - 1, i);
```

Comment:
Please take latest btree_benchmark code and use cr_btree_key_make() for creating keys . You can fill sequential keys for now using cwb->cwb_bo[BOT_INSERT].key as key,

Reply:
Ok. I will do update it!